cmd/metal-api/internal/service/v1/machine.go

```diff
@@ -389,9 +378,16 @@ func NewMachineResponse(m *metal.Machine, s *metal.Size, p *metal.Partition, i *
 
 	nics := MachineNics{}
 	for i := range m.Hardware.Nics {
+		n := m.Hardware.Nics[i]
+		neighs := MachineNics{}
+		for j := range n.Neighbors {
+			neigh := n.Neighbors[j]
+			neighs = append(neighs, MachineNic{MacAddress: string(neigh.MacAddress), Name: neigh.Name})
+		}
```

Comment:
How did we do that before with the extended struct?

Reply:
This was not done before because, this was a BUG?